package.json

```diff
@@ -1,18 +1,25 @@
 {
-  "name": "webrtc-signal-http",
-  "version": "1.7.0",
+  "name": "webrtc-signal-http-ts",
+  "version": "1.8.22",
```

Comment:
why .22?

Reply:
there were several attempts to integrate the type scripted webrtc-signal-http into the larger signaler, and for each, I published another version of the package 😁

Reply:
oh i see what's happened here. we should put the name back and version to 2.0 or 1.8.0 - thanks!